util/insert.go

```diff
@@ -0,0 +1,111 @@
+package util
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/bmeg/grip/gripql"
+	"github.com/bmeg/grip/log"
+	"github.com/hashicorp/go-multierror"
+)
+
+// StreamBatch a stream of inputs and loads them into the graph
+// This function assumes incoming stream is GraphElemnts from a single graph
+func StreamBatch(stream <-chan *gripql.GraphElement, batchSize int, graph string, vertexAdd func([]*gripql.Vertex) error, edgeAdd func([]*gripql.Edge) error) error {
```

Comment:
It might make more sense to use key/value serialized value size to determine batches, rather then use a batchSize. But that can probably left for another PR.

Reply:
Agreed.